examples/svdiff/requirements.txt

```diff
@@ -0,0 +1,8 @@
+accelerate>=0.16.0
+torchvision
+transformers>=4.25.1
+safetensors
+ftfy
+tensorboard
+Jinja2
+einops
```

Comment:
We need to try to eliminate this dependency and resort to native PyTorch operations here for einsum stuff.